seed/views/v3/columns.py

```diff
@@ -106,7 +103,7 @@ def list(self, request):
         only_used = json.loads(request.query_params.get('only_used', 'false'))
         columns = Column.retrieve_all(organization_id, inventory_type, only_used)
         organization = Organization.objects.get(pk=organization_id)
-        if json.loads(request.query_params.get('display_units', 'false')):
```

Comment:
👍 
The only thing we'd have to change is on the frontend. Specifically on the organization column settings page, we shouldn't see units appended to display names as the display names are editable.
<img width="1113" alt="Screen Shot 2020-11-12 at 3 39 11 PM" src="https://user-images.githubusercontent.com/30608004/98994094-4e111400-24fd-11eb-896e-f664dee8d902.png">

Reply:
<img width="561" alt="Screen Shot 2020-11-12 at 4 46 24 PM" src="https://user-images.githubusercontent.com/564231/99010210-c203ea00-2506-11eb-9726-209160bafc6d.png">
<img width="882" alt="Screen Shot 2020-11-12 at 4 47 08 PM" src="https://user-images.githubusercontent.com/564231/99010212-c3cdad80-2506-11eb-8f96-11a6372a61f5.png">
Note that display units is false for the settings page and true for the inventory list.